conda_forge_metadata/feedstock_outputs.py

```diff
@@ -0,0 +1,46 @@
+from functools import lru_cache
+import requests
+
+
+@lru_cache(maxsize=1024)
+def package_to_feedstock(name):
+    """Map a package name to the feedstock name.
+    Parameters
+    ----------
+    package : str
+        The name of the package.
+    Returns
+    -------
+    feedstock : str
+        The name of the feedstock, without the ``-feedstock`` suffix.
+    """
+    assert name, "name must not be empty"
+
+    # See https://github.com/conda-forge/feedstock-outputs/blob/c35451f2fb8b7/scripts/shard_repo.py
+    # for sharding details.
+    shard_size = 3
```

Comment:
We should put this in a json blob in the feedstock-outputs repo so that if it ever changes, old versions of this package still work.

Reply:
Excellent idea. I'll add a param file right away.

Reply:
https://github.com/conda-forge/feedstock-outputs/pull/44

Reply:
Awesome. We should move to this file here too: https://github.com/conda-forge/conda-forge-webservices/blob/main/conda_forge_webservices/feedstock_outputs.py#L29. No need to do it now.

Reply:
And there is code in the cf ci setup package.

Reply:
Should we move those to using `conda-forge-metadata` instead? I can happily add the same changes there but it feels like duplicated code.

Reply:
Or maybe that's what you always meant hah

Reply:
I don't care either way as long as things are stable.

Reply:
Added two draft PRs:

- https://github.com/conda-forge/conda-forge-webservices/pull/552
- https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/233